internal/util/plugins/makefile.go

```diff
@@ -97,5 +98,11 @@ bundle: kustomize
 # Build the bundle image.
 bundle-build:
 	docker build -f bundle.Dockerfile -t $(BUNDLE_IMG) .
+`
+	//todo(camilamcedo86): remove it when we get v3 plugin (it is implemented there)
+	makefileUndeployFragmentGo = `
+# UnDeploy controller from the configured Kubernetes cluster in ~/.kube/config
+undeploy:
+	$(KUSTOMIZE) build config/default | kubectl delete -f -
```

Comment:
IMO as a follow up we need to: 
- Add in the plugins a dir such we have now the scorecard to have the olm postScafffold actions (update makefile to add its subcommands)
- And then, another one just for append SDK utils as this one.